diff --git a/node_modules/@types/react-native/index.d.ts b/node_modules/@types/react-native/index.d.ts
index c20abbb..f77757c 100755
--- a/node_modules/@types/react-native/index.d.ts
+++ b/node_modules/@types/react-native/index.d.ts
@@ -7044,6 +7044,7 @@ export type ShareOptions = {
     excludedActivityTypes?: Array<string> | undefined;
     tintColor?: ColorValue | undefined;
     subject?: string | undefined;
+    anchor?: number | undefined;
 };
 
 export type ShareSharedAction = {
