diff --git a/node_modules/native-base/lib/commonjs/components/primitives/Input/Input.js b/node_modules/native-base/lib/commonjs/components/primitives/Input/Input.js
index 67d7dd2..4525de9 100644
--- a/node_modules/native-base/lib/commonjs/components/primitives/Input/Input.js
+++ b/node_modules/native-base/lib/commonjs/components/primitives/Input/Input.js
@@ -80,7 +80,7 @@ const Input = ({
     isInvalid: inputThemeProps.isInvalid,
     isReadOnly: inputThemeProps.isReadOnly
   });
-  const [layoutProps, nonLayoutProps] = (0, _utils.extractInObject)(resolvedProps, [..._utils.stylingProps.margin, ..._utils.stylingProps.border, ..._utils.stylingProps.layout, ..._utils.stylingProps.flexbox, ..._utils.stylingProps.position, ..._utils.stylingProps.background, 'shadow', 'opacity']);
+  const [layoutProps, nonLayoutProps] = (0, _utils.extractInObject)(resolvedProps, [..._utils.stylingProps.margin, ..._utils.stylingProps.border, ..._utils.stylingProps.layout, ..._utils.stylingProps.flexbox, ..._utils.stylingProps.position, ..._utils.stylingProps.background, 'shadow', 'opacity', 'boxShadow']);
   const [, baseInputProps] = (0, _utils.extractInObject)(nonLayoutProps, ['variant']); //TODO: refactor for responsive prop
 
   if ((0, _useHasResponsiveProps.useHasResponsiveProps)(props)) {
diff --git a/node_modules/native-base/lib/commonjs/theme/components/icon-button.js b/node_modules/native-base/lib/commonjs/theme/components/icon-button.js
index c7378bc..85664f8 100644
--- a/node_modules/native-base/lib/commonjs/theme/components/icon-button.js
+++ b/node_modules/native-base/lib/commonjs/theme/components/icon-button.js
@@ -19,7 +19,12 @@ const baseStyle = props => {
   return {
     borderRadius: 'sm',
     _web: {
-      cursor: props.isDisabled ? 'not-allowed' : props.isLoading ? 'default' : 'pointer'
+      _disabled: {
+        cursor: 'not-allowed'
+      },
+      _loading: {
+        cursor: 'not-allowed'
+      },
     },
     _focusVisible: {
       style: props.variant !== 'unstyled' ? { ...focusRing
diff --git a/node_modules/native-base/lib/commonjs/theme/styled-system.js b/node_modules/native-base/lib/commonjs/theme/styled-system.js
index 97309ef..78e3a8d 100644
--- a/node_modules/native-base/lib/commonjs/theme/styled-system.js
+++ b/node_modules/native-base/lib/commonjs/theme/styled-system.js
@@ -576,6 +576,7 @@ exports.typography = typography;
 const extraProps = {
   outline: true,
   outlineWidth: true,
+  boxShadow: true,
   shadow: {
     scale: 'shadows'
   },
diff --git a/node_modules/native-base/lib/module/components/primitives/Input/Input.js b/node_modules/native-base/lib/module/components/primitives/Input/Input.js
index cdd877c..a3111cf 100644
--- a/node_modules/native-base/lib/module/components/primitives/Input/Input.js
+++ b/node_modules/native-base/lib/module/components/primitives/Input/Input.js
@@ -58,7 +58,7 @@ const Input = ({
     isInvalid: inputThemeProps.isInvalid,
     isReadOnly: inputThemeProps.isReadOnly
   });
-  const [layoutProps, nonLayoutProps] = extractInObject(resolvedProps, [...stylingProps.margin, ...stylingProps.border, ...stylingProps.layout, ...stylingProps.flexbox, ...stylingProps.position, ...stylingProps.background, 'shadow', 'opacity']);
+  const [layoutProps, nonLayoutProps] = extractInObject(resolvedProps, [...stylingProps.margin, ...stylingProps.border, ...stylingProps.layout, ...stylingProps.flexbox, ...stylingProps.position, ...stylingProps.background, 'shadow', 'opacity', 'boxShadow']);
   const [, baseInputProps] = extractInObject(nonLayoutProps, ['variant']); //TODO: refactor for responsive prop
 
   if (useHasResponsiveProps(props)) {
diff --git a/node_modules/native-base/lib/module/theme/components/icon-button.js b/node_modules/native-base/lib/module/theme/components/icon-button.js
index 7515355..5c46602 100644
--- a/node_modules/native-base/lib/module/theme/components/icon-button.js
+++ b/node_modules/native-base/lib/module/theme/components/icon-button.js
@@ -11,7 +11,12 @@ const baseStyle = props => {
   return {
     borderRadius: 'sm',
     _web: {
-      cursor: props.isDisabled ? 'not-allowed' : props.isLoading ? 'default' : 'pointer'
+      _disabled: {
+        cursor: 'not-allowed'
+      },
+      _loading: {
+        cursor: 'not-allowed'
+      },
     },
     _focusVisible: {
       style: props.variant !== 'unstyled' ? { ...focusRing
diff --git a/node_modules/native-base/lib/module/theme/styled-system.js b/node_modules/native-base/lib/module/theme/styled-system.js
index 637de0c..715ce46 100644
--- a/node_modules/native-base/lib/module/theme/styled-system.js
+++ b/node_modules/native-base/lib/module/theme/styled-system.js
@@ -553,6 +553,7 @@ export const typography = {
 const extraProps = {
   outline: true,
   outlineWidth: true,
+  boxShadow: true,
   shadow: {
     scale: 'shadows'
   },
diff --git a/node_modules/native-base/src/components/composites/AlertDialog/AlertDialogBody.tsx b/node_modules/native-base/src/components/composites/AlertDialog/AlertDialogBody.tsx
index 10243c4..27d5151 100644
--- a/node_modules/native-base/src/components/composites/AlertDialog/AlertDialogBody.tsx
+++ b/node_modules/native-base/src/components/composites/AlertDialog/AlertDialogBody.tsx
@@ -11,8 +11,8 @@ const AlertDialogBody = (props: IBoxProps, ref?: any) => {
     return null;
   }
   return (
-    <ScrollView>
-      <Box {...newProps} ref={ref}>
+    <ScrollView ref={ref}>
+      <Box {...newProps}>
         {props.children}
       </Box>
     </ScrollView>
diff --git a/node_modules/native-base/src/components/primitives/Input/Input.tsx b/node_modules/native-base/src/components/primitives/Input/Input.tsx
index 4adb61f..d8df714 100644
--- a/node_modules/native-base/src/components/primitives/Input/Input.tsx
+++ b/node_modules/native-base/src/components/primitives/Input/Input.tsx
@@ -69,6 +69,7 @@ const Input = (
     ...stylingProps.background,
     'shadow',
     'opacity',
+    'boxShadow',
   ]);
 
   const [, baseInputProps] = extractInObject(nonLayoutProps, ['variant']);
diff --git a/node_modules/native-base/src/theme/components/icon-button.ts b/node_modules/native-base/src/theme/components/icon-button.ts
index 7e30a1f..9d2ba52 100644
--- a/node_modules/native-base/src/theme/components/icon-button.ts
+++ b/node_modules/native-base/src/theme/components/icon-button.ts
@@ -11,11 +11,12 @@ const baseStyle = (props: any) => {
   return {
     borderRadius: 'sm',
     _web: {
-      cursor: props.isDisabled
-        ? 'not-allowed'
-        : props.isLoading
-        ? 'default'
-        : 'pointer',
+      _disabled: {
+        cursor: 'not-allowed'
+      },
+      _loading: {
+        cursor: 'not-allowed'
+      },
     },
     _focusVisible: {
       style: props.variant !== 'unstyled' ? { ...focusRing } : {},
diff --git a/node_modules/native-base/src/theme/styled-system.ts b/node_modules/native-base/src/theme/styled-system.ts
index 6dc36a5..c27b9fc 100644
--- a/node_modules/native-base/src/theme/styled-system.ts
+++ b/node_modules/native-base/src/theme/styled-system.ts
@@ -566,6 +566,7 @@ export const typography = {
 const extraProps = {
   outline: true,
   outlineWidth: true,
+  boxShadow: true,
   shadow: {
     scale: 'shadows',
   },
